src/SonataCoreBundle.php

```diff
@@ -125,7 +124,6 @@ public function registerFormMapping()
             'sonata_type_immutable_array' => ImmutableArrayType::class,
             'sonata_type_boolean' => BooleanType::class,
             'sonata_type_collection' => CollectionType::class,
-            'sonata_type_translatable_choice' => TranslatableChoiceType::class,
```

Comment:
Isnt this a BC break? what if I am using this type?

Reply:
Removed this